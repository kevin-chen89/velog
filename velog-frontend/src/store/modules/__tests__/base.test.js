// // @flow
// import invariant from 'invariant';
// import base, { actionCreators, type Base, BaseActionCreators } from '../base';


// describe('base', () => {
//   describe('reducer', () => {
//     let state: Base = base((undefined: any), ({}: any));

//     it('initializes state', () => {
//       expect(state).toMatchSnapshot();
//     });
//     it('action: SHOW_USER_MENU', () => {
//       if (!state) return;
//       state = base(state, actionCreators.showUserMenu());
//     });
//     it('action: HIDE_USER_MENU', () => {
//       if (!state) return;
//       state = base(state, actionCreators.hideUserMenu());
//     });
//     it('action: ', () => {
//       if (!state) return;
//       state = base(state, actionCreators.setFullscreenLoader(true));
//     });
//   });
// });