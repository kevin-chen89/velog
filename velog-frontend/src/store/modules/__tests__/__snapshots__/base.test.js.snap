// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`base reducer initializes state 1`] = `
Object {
  "fullscreenLoader": false,
  "userMenu": false,
}
`;
